gutenberg.php

```diff
@@ -34,7 +34,7 @@ function the_gutenberg_project() {
 		<?php
 		printf(
 			/* translators: %s: https://wordpress.org/plugins/classic-editor/ */
-			__( 'The Block Editor requires JavaScript. Please try the <a href="%s">Classic Editor plugin</a>.', 'gutenberg' ),
+			__( 'The block editor requires JavaScript. Please try the <a href="%s">Classic Editor plugin</a>.', 'gutenberg' ),
```

Comment:
For posterity's sake: This change resulted in a conflict for #13569 , which prompted me to confirm that it is updated as such already in core. It appears to be already correct there:

https://github.com/WordPress/wordpress-develop/blob/91e5365/src/wp-admin/edit-form-blocks.php#L412

Reply:
Thanks @aduth should this be retracked from this PR or is there any actions I need to take to address that conflict? Appreciated

Reply:
Nope, there's nothing which needs to be done. It's still planned to be removed in #13569, but I just wanted to make sure the equivalent code is correct in core. It appears it's already correct 👍

Reply:
Ah gotcha, thanks for clarifying.